platform.md

```diff
@@ -122,6 +125,13 @@ The buildpacks directory MUST contain unarchived buildpacks such that:
 - Each top-level directory is a buildpack ID.
 - Each second-level directory is a buildpack version.
 
+## OCI Image
+The OCI image containing the compiled application is generated by the Build Image during the [Build Phase](#build-phase).
+
+If the `/cnb/lifecycle/builder` is invoked with the optional `-project` argument, the lifecycle MUST add a `"io.buildpacks.project"` Config label to the new OCI image with a [value as described in the data-format](#io.buildpacks.app.source-oci-image-label) section.
```

Comment:
Stylistic things:
- Use `io.buildpacks.project` without quotes to match other labels.
- "Config" -> "image config" to match other text.

Reply:
Changes are now incorporated in the latest commit.